connections/jest.config.js

```diff
@@ -0,0 +1,8 @@
+// jest.config.js (javascript file)
+module.exports = {
+    preset: 'jest-preset-angular',
+    setupFilesAfterEnv: ['<rootDir>/setup-jest.ts'],
```

Comment:
@Siddharth-Tiwari1712 can you check with Sunny, about how we have it set up in Biotool. I think I have it brokenly set up like this is a jest config. Except. I think we should be remvong it, in favor of the one at the bottom of Package.json.
just check with him

Reply:
I checked one article.
This makes it easier to use absolute paths in your imports, avoiding the need for long relative paths like ../../../.
Since all spec files had src in path i used this.

We can surely remove this I'll ask sunny once.